```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/travis/build/opencast/opencast/modules/kernel/src/main/java/org/opencastproject/kernel/security/CustomPasswordEncoder.java:[78,14] cannot find symbol
  symbol:   variable StringUtils
  location: class org.opencastproject.kernel.security.CustomPasswordEncoder
[INFO] 1 error
```

You need an import if you want to use this.

--- a/modules/kernel/src/main/java/org/opencastproject/kernel/security/CustomPasswordEncoder.java
+++ b/modules/kernel/src/main/java/org/opencastproject/kernel/security/CustomPasswordEncoder.java
@@ -22,6 +22,7 @@
 package org.opencastproject.kernel.security;
 
 import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.security.authentication.encoding.PasswordEncoder;
@@ -75,7 +76,7 @@ public class CustomPasswordEncoder implements PasswordEncoder {
     // Test BCrypt encoded hash
     logger.debug("Verifying bcrypt hash {}", encodedPassword);
     try {
-      return encodedPassword.charAt(0) == '$' && BCrypt.checkpw(rawPassword, encodedPassword);
+      return StringUtils.startsWith(encodedPassword, "$") && BCrypt.checkpw(rawPassword, encodedPassword);
     } catch (IllegalArgumentException e) {
       logger.debug("bcrypt hash verification failed", e);
     }

